docs/en/00_Getting_Started/02_Composer.md

```diff
@@ -56,48 +64,57 @@ see [Using development versions](#using-development-versions).
+`master` is not a legal version string - it's a branch name. These are different things. The version string that would get you the branch is `dev-master`. The version string that would get you a numeric branch is a little different. The version string for the `3` branch is `3.x-dev`.
 </div>
 
 ## Updating dependencies
 
-Except for the control code of the Voyager space probe, every piece of code in the universe gets updated from time to time.  SilverStripe modules are no exception.
+Except for the control code of the Voyager space probe, every piece of code in the universe gets updated from time to time. SilverStripe modules are no exception.
 
 To get the latest updates of the modules in your project, run this command:
 
-	composer update --no-dev
+```
+composer update --no-dev
+```
 
-Updates to the required modules will be installed, and the `composer.lock` file will get updated with the specific commits of each of those.
+Updates to the required modules will be installed, and the `composer.lock` file will get updated with the specific commits and version constraints for each of them.
 
 ## Deploying projects with Composer
 
-When deploying projects with composer, you could just push the code and run `composer update`.  However, this is risky.  In particular, if you were referencing development dependencies and a change was made between your testing and your depoyment to production, you would end up deploying untested code.  Not cool!
+When deploying projects with composer, you could just push the code and run `composer update`. This, however, is risky. In particular, if you were referencing development dependencies and a change was made between your testing and your deployment to production, you would end up deploying untested code. Not cool!
 
-The `composer.lock` file helps with this.  It references the specific commits that have been checked out, rather than the version string.  You can run `composer install` to install dependencies from this rather than `composer.json`.
+The `composer.lock` file helps with this. It references the specific commits that have been checked out, rather than the version string. You can run `composer install` to install dependencies from this rather than `composer.json`.
 
-So, your deployment process, as it relates to Composer, should be as follows:
+So your deployment process, as it relates to Composer, should be as follows:
 
- * Run `composer update` on your development version before you start whatever testing you have planned.  Perform all the necessary testing.
+ * Run `composer update` on your development version before you start whatever testing you have planned. Perform all the necessary testing.
  * Check `composer.lock` into your repository.
  * Deploy your project code base, using the deployment tool of your choice.
  * Run `composer install --no-dev -o` on your production version.
```

Comment:
I might defer this one to @tractorcow. I'm not sure whether it would be considered as a "recommended practice" by us or not, but I'm sure he will :)